pkg/models/apiv2.go

```diff
@@ -169,6 +169,11 @@ type AddonBundle struct {
 	Options *BundleOptions `json:"options"`
 }
 
+type Credentials struct {
+	Username string
+	Password string
+}
+
```

Comment:
This is more of an operator model rather than apiv2.

Reply:
fixed